ocs_ci/ocs/registry.py

```diff
@@ -279,3 +291,19 @@ def image_rm(registry_path):
     ocp_obj = ocp.OCP()
     ocp_obj.exec_oc_debug_cmd(node=master_list[0], cmd_list=cmd_list)
     logger.info(f"Image {registry_path} rm successful")
+
+
+def check_image_in_registry(image_url):
```

Comment:
sorry for the late comments, but its better if you can rename this as check_image_exists since its raising an exception and needs Returns: in docstring

Reply:
@vasukulkarni Will update doc string of function with Raises.
> + needs Returns
Do you want returns to be added to this func?  I removed it due to the exception and had only return True statement. Let me know if I misunderstood it.

Reply:
@vasukulkarni updated the code.